src/components/learner-credit-management/data/utils.js

```diff
@@ -570,6 +570,29 @@ export const minimumEnrollByDateFromToday = ({ subsidyExpirationDatetime }) => M
   dayjs(subsidyExpirationDatetime).subtract(MAX_ALLOWABLE_REFUND_THRESHOLD_DAYS, 'days').toDate(),
 );
 
+const subsidyExpirationRefundCutoffDate = ({ subsidyExpirationDatetime }) => dayjs(subsidyExpirationDatetime).subtract(MAX_ALLOWABLE_REFUND_THRESHOLD_DAYS, 'days').toDate();
```

Comment:
nit: you could re-use this directly above in `minimumEnrollByDateFromToday`. Actually now that I'm looking at that function, is the `min()` doing anything?

Reply:
I believe the `min` portion was left in incase additional criteria would need to be added, for right now its determining a fallback enrollby date to display within `getNormalizedEnrollByDate`.